src/SDKs/Azure.Core/data-plane/Azure.Core/Pipeline/Policies/ExponentialRetryPolicy.cs

```diff
@@ -29,6 +29,12 @@ public ExponentialRetryPolicy()
 
         protected override void GetDelay(HttpPipelineMessage message, int attempted, out TimeSpan delay)
         {
+            base.GetDelay(message, attempted, out delay);
```

Comment:
Normally, in an exponential back-off, the pattern that I've seen (and used) is for any retry delay specified by the service to be added to the current exponential back off time, rather than supersede it.   In the case where we, for some reason, don't want them to compound, I believe that we would want to take the longer of the two, no?